interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
@robertogagliotta ,
Note these settings were set in OpenEMR's stylesheets previously and they were clashing with bootstrap's nice alert classes. This is why it was renamed to alert-custom along with several places in the codebase that are using them (and are places where bootstrap is not yet used).